src/recommendation/src/main/scala/recommendation.txt

```diff
@@ -0,0 +1 @@
+Recommender Software.
```

Comment:
you need to add a summary here in the style of other modules.  This is what is shown in the python/R docs for the module.